src/org/omegat/core/data/ExternalTMFactory.java

```diff
@@ -168,17 +166,17 @@ private void addTuv(TMXReader2.ParsedTu tu, TMXReader2.ParsedTuv tuvSource,
                     for (int i = 0; i < sources.size(); i++) {
                         PrepareTMXEntry te = new PrepareTMXEntry();
                         te.source = sources.get(i);
```

Comment:
Is it necessary to add `PrepareTMXEntry#setSource`, `setCreator`, and `SetCreationDate` method?